doc/sphinx/source/recipes/recipe_kcs.rst

```diff
@@ -0,0 +1,142 @@
+
+The original method created 16 resampled datasets:
+
+* 2 main scenarios: Moderate (M) and Warm (W) (Lenderink 2014 uses "G" instead of "M").
+* 2 'sub'scenarios: Relatively high (H) or low (L) changes in seasonal temperature and precipitation
+* 2 time horizons: Mid-century (MOC; 2050) and end-of-century (EOC; 2085)
+* 2 periods: Control (1981-2010) and future (variable)
+
+The configuration settings for these scenarios can be found in table 1 of Lenderink 2014's supplement.
+
+Implementation
+--------------
+
+The implementation is such that application to other datasets, regions, etc. is relatively straightforward. However, the description below focuses on the reference use case of Lenderink et al., 2014, where the target model is EC-Earth.
+
+In the first diagnostic, the spread of the full CMIP ensemble is used to obtain 4 values of a *global* :math:`{\Delta}T_{CMIP}`, corresponding to the 10th and 90th percentiles for the M and W scenarios, respectively, for both MOC and EOC. Subsequently, for each of these 4 *steering parameters*, 30-year periods are selected from the EC-Earth ensemble, where :math:`{\Delta}T_{ECEarth}{\approx}{\Delta}T_{CMIP}`.
+
+In the second diagnostic, for for both the control and future periods, the 8 EC-Earth ensemble members are split into 6 segments of 5 years each. Out of all :math:`8^6` possible re-combinations of these 5-year segments, eventually 8 new 'resamples' are selected based on *local* changes in seasonal temperature and precipitation. This is done in the following steps:
+
+1. Select 1000 samples for the control period, and 2 x 1000 samples for the future period (one for each subscenario). Step 1 poses a constraint on winter precipitation. For the control period, winter precipitation must still closely represent the average of the original ensemble. For the two future periods, the change in winter precipitation with respect to the control period must approximately equal :math:`4{\Delta}T` % (subscenario L) or  :math:`8{\Delta}T` % (subscenario H).
+2. Further constrain the selection by picking samples that represent either high or low changes in summer precipitation and summer and winter temperature, by limiting the remaining samples to certain percentile ranges: relatively wet/cold in the control and dry/warm in the future, or vice versa. The percentile ranges are listed in table 1 of Lenderink 2014's supplement. This should result is approximately 50 remaining samples for each scenario, for both control and future.
+3. Use a monte-carlo method to make a final selection of 8 resamples with minimal reuse of the same ensemble member/segment.
+
+
+.. note::
```

Comment:
the section "Available recipes and diagnostics" might be a better place to have this note.

Reply:
I'll look into that.

Reply:
Done